debian/postinst

```diff
@@ -68,12 +68,32 @@ print(*ENTITLEMENT_CLASS_BY_NAME.keys(), sep=' ')
     done
 }
 
-check_esm_infra_is_enabled() {
-    ua status | egrep -q 'esm-infra.*enabled' && return 0 || return 1
+
+# Check cached service status from status.json and return 0 if enabled else 1
+check_service_is_enabled() {
+    service_name=$1
+    _RET=$(python3 -c "
+import os
+import json
+
+status_file = '/var/lib/ubuntu-advantage/status.json'
```

Comment:
I think it would be better if we use `UAConfig` to access the status cache. By doing that, I think we can avoid checking if the file exists, for example

Reply:
done